### ISSUE TITLE
Leave default Ubuntu sources.list

### ISSUE THREAD

**Issue by lisotton** [ciandt-dev/docker-hub-php#29]
Although the performance of closest repositories are better sometimes they doesn't work as expected.

**Comment by ivan-pinatti** [comment]
@lisotton did you check the DNS set at the moment ?

VPNs can be trick with the DNS, you should force to always use the closest to where you are executing docker build and run.
I would set the DNS for Docker service to fix BUILD issues and at docker run / docker compose to fix RUN issues.

**Comment by lisotton** [comment]
The last time that I tested here in Italy, some of repositories are broken and then reverting to default repositories it works fine.

**Comment by ivan-pinatti** [comment]
@lisotton maybe you ran when the repos were synchronizing, could you try again sometime ?

**Comment by lisotton** [comment]
Yes, I'll try it, but I already faced this issue multiple times in Brazil also.

**Comment by ivan-pinatti** [comment]
Actually, when #27 is implemented it can be removed because it is the default behavior in the new Ubuntu version.

@erickj-ciandt can you take care of it?
